public/app/plugins/panel/heatmap/migrations.ts

```diff
@@ -84,7 +86,7 @@ export function angularToReactHeatmap(angular: any): { fieldConfig: FieldConfigS
     calculate,
     calculation,
     color: {
-      ...defaultPanelOptions.color,
+      ...defaultPanelOptions.color!,
```

Comment:
```suggestion
      ...defaultPanelOptions.color,
```
Since we re-exported defaultOptions without optional members, these assertions are not needed anymore.